File: VB/E4422/Program.vb
Imports DevExpress.XtraRichEdit
Imports System
Imports System.Windows.Forms

Namespace E4422

    Friend Module Program

        ''' <summary>
        ''' The main entry point for the application.
        ''' </summary>
        <STAThread>
        Sub Main()
            Call Application.EnableVisualStyles()
            Application.SetCompatibleTextRenderingDefault(False)
            RichEditControlCompatibility.UseThemeFonts = False
            Call Application.Run(New Form1())
        End Sub
    End Module
End Namespace

File: CS/E4422/Program.cs
using DevExpress.XtraRichEdit;
using System;
using System.Windows.Forms;

namespace E4422 {
    static class Program {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main() {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            RichEditControlCompatibility.UseThemeFonts = false;
            Application.Run(new Form1());
        }
    }
}
